guide/commando/args.md

```diff
@@ -57,13 +57,13 @@ See? Simple.
 
 - `key` is the name of the argument. When you define it in your `run` method, this is what you'll be using.
 - `prompt` is the text that displays if no argument is provided. If someone uses just `?say`, that prompt will come up asking for the text.
-- `type` is the type the argument is a part of. This can be many things, including `string`, `integer`, `user`, `member`, etc.
+- `type` is the type the argument is a part of. This can be many things, including `string`, `integer`, `user`, 'member`, etc.
```

Comment:
```suggestion
- `type` is the type the argument is a part of. This can be many things, including `string`, `integer`, `user`, `member`, etc.
```
Revert this change and others on the page.